Frontend/DAT257_Website/src/Components/Header/Header.css

```diff
@@ -0,0 +1,89 @@
+.Header {
```

Comment:
lowercase h

Reply:
The Header className is for convention named Header, I can't use lwoer case h if I want to access it. Same goes for .DonationBox